app/routes.py

```diff
@@ -145,7 +145,8 @@ def search(group_id):
             isochrones, best_places = matching.match_bars(recipients_enriched, get_database_uri(**credentials['db']), limit=3)
             session['results'] = best_places
             # GENERATE MAP
-            map_html_path = 'app/templates/map.html'
+            map_html_path = f'app/templates/maps/map_{group_id}.html'
+            print(1, map_html_path )
```

Comment:
I guess we should not leave prints in the code

Reply:
True, thanks!